src/fev/utils.py

```diff
@@ -181,3 +182,23 @@ def process_entry(group: pd.DataFrame) -> dict:
     with mp.Pool(processes=num_proc) as pool:
         entries = pool.map(process_entry, [group for _, group in df.groupby(id_column, sort=False)])
     return datasets.Dataset.from_list(entries)
+
+
+def generate_fingerprint(dataset: datasets.Dataset) -> str | None:
```

Comment:
@abdulfatir If this logic looks good to you, I will also update the fingerprints in the saved [evaluation results](https://github.com/autogluon/fev/tree/main/benchmarks/chronos_zeroshot/results) on GitHub